src/core/utils/checks/charge_neutrality.hpp

```diff
@@ -0,0 +1,33 @@
+#include <cmath>
+
+#include "PartCfg.hpp"
+
+namespace Utils {
+
+inline void check_charge_neutrality(PartCfg &partCfg) {
```

Comment:
Make the particle range a template parameter so that the range can be used on any range type.